### PR TITLE
Update cloud plans data

### PR DESCRIPTION
**Pull request by mdubus**

### CODE REVIEW

File: data/pricing.js [hunk 1]
```diff
@@ -34,41 +39,94 @@ const getPricingPageData = () => ({
...
+        specifications: [
+          'Disk 10GB',
+          'RAM 1GB',
+          '0,5vCPU',
+          'Shared resources',
+          'Community (Slack) support',
+        ],
+        cta: {
+          title: 'Get started',
+          href: '', // TODO: ask for the link
```
Comment (bidoubiwa) [inline data/pricing.js:57]: Is this for later?

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥

**Comment by mdubus** [comment]
bors merge

**Comment by mdubus** [comment]
bors cancel

**Pull request merged**